slimta/edge/wsgi.py

```diff
@@ -89,13 +90,25 @@ class WsgiResponse(Exception):
 
     """
 
-    def __init__(self, status, headers=None, data=None):
+    def __init__(self, status, headers=[], data=[]):
```

Comment:
I still have a thing with putting `[]` or `{}` as default argument values because of [this old gotcha](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments). In this particular case, it won't cause a problem since you never mutate the default, but it's still something I consider "bad practice".